CHARTER.md

```diff
@@ -0,0 +1,89 @@
+
+### **1.1: In-scope**
+
+- GPU-accelerated graph algorithms  
+- WebGL-based rendering of large-scale graph structures  
+- Browser-native integration with frontend tooling and workflows
+
+### **1.2: Out-of-Scope**
+
+- Full-fledged graph analytics  
+- Backend data processing and pipelines
+
+## **Section 2: Relationship with OpenJS Foundation CPC**
+
+Technical leadership of the cosmos.gl project is delegated to the cosmos.gl Technical Steering Committee (TSC) by the OpenJS Cross Project Council (CPC). The TSC is also responsible for deciding how to allocate earmarked funds it might receive through the Foundation. Amendments to this charter require approval from both the CPC, through its [decision-making process](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-9-decision-making), and the TSC.
+
+### **2.1 Other Formal Project Relationships**
+
+Section Intentionally Left Blank.
+
+## **Section 3: cosmos.gl TSC Governing Body**
+
+TSC members can be either regular members or voting members. Regular members can attend meetings and participate in TSC discussions, but do not vote. Voting members can do everything regular members can do, and also have the ability to cast votes when consensus is not reached on an issue.
+
+TSC memberships are not time-limited. There is no maximum size of the TSC. 
+
+There is no specific set of requirements or qualifications for TSC membership beyond these rules. The TSC may add additional voting members to the TSC by a standard TSC motion. A TSC member can be removed from the TSC by voluntary resignation or by a standard TSC motion. A standard TSC motion can be used to change a regular TSC member to a voting TSC member, or to change a voting TSC member to a regular TSC member.
+
+The TSC shall meet regularly using tools that enable participation by the community. The meeting shall be directed by the TSC Chairperson. Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+TSC voting members are expected to regularly participate in TSC activities.
+
+A TSC voting member is automatically converted to a TSC regular member if they do not participate in three consecutive TSC votes.
+
+## **Section 4: Roles & Responsibilities**
+
+Subject to such policies as may be set by the CPC, the TSC is responsible for all technical development within the [cosmos.gl](http://cosmos.gl)  project, including:
+
+- Setting release dates.  
+- Release quality standards.  
+- Technical direction.  
+- Project governance and process.  
+- GitHub repository hosting.  
+- Conduct guidelines.  
+- Maintaining the list of additional Collaborators.  
```

Comment:
If you capitalize terms, one would expect a definition somewhere. Just use lowercase instead.